### PR TITLE
Added an Helm Chart

### PR DESCRIPTION
**Pull request by l0rd**
### What does this PR do?

Currently the README.md had only instructions to deploy the registry on Docker and OpenShift. This PR adds an Helm Chart and the instructions to use it to deploy the registry on Kubernetes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,31 @@ You can deploy Che plugin registry on Openshift with command.
...
 ```
+
+## Kubernetes
+
+You can deploy Che plugin registry on Kubernetes using [helm](https://docs.helm.sh/).
+
+```bash
+
+NAMESPACE="kube-che"
+DOMAIN="$(minikube ip).nip.io"
```
Comment (benoitf) [inline README.md]: here it looks like more for minikube than k8s

File: kubernetes/che-plugin-registry/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2012-2017 Red Hat, Inc.
```
Comment (benoitf) [inline kubernetes/che-plugin-registry/Chart.yaml]: invalid copyright date

File: kubernetes/che-plugin-registry/templates/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2012-2017 Red Hat, Inc.
```
Comment (benoitf) [inline kubernetes/che-plugin-registry/templates/ingress.yaml]: Date should be 2018 (btw in 2012 it was not also started)

File: kubernetes/che-plugin-registry/values.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2012-2017 Red Hat, Inc.
```
Comment (benoitf) [inline kubernetes/che-plugin-registry/values.yaml]: date

File: kubernetes/che-plugin-registry/values.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+chePluginRegistryImage: eclipse/che-plugin-registry
+chePluginRegistryImagePullPolicy: Always
+chePluginRegistryMemoryLimit: 256Mi
+
+global:
+  ingressDomain: 192.168.99.100.nip.io
```
Comment (benoitf) [inline kubernetes/che-plugin-registry/values.yaml]: IP is hardcoded ?

(Missing also a newline)

File: README.md (hunk 1 above)
Comment (l0rd) [reply README.md]: I have updated the sentence above to make it clear that in the example below the target platform is minikube.

File: kubernetes/che-plugin-registry/Chart.yaml (hunk 2 above)
Comment (l0rd) [reply kubernetes/che-plugin-registry/Chart.yaml]: fixed

File: kubernetes/che-plugin-registry/templates/ingress.yaml (hunk 3 above)
Comment (l0rd) [reply kubernetes/che-plugin-registry/templates/ingress.yaml]: fixed

File: kubernetes/che-plugin-registry/values.yaml (hunk 4 above)
Comment (l0rd) [reply kubernetes/che-plugin-registry/values.yaml]: fixed

File: kubernetes/che-plugin-registry/values.yaml (hunk 5 above)
Comment (l0rd) [reply kubernetes/che-plugin-registry/values.yaml]: Added the new line. For the ingressDomain default value I am using the hardcoded IP we use in Che Server Helm chart. We still haven't found a better solution.

**Pull request merged**